### PR TITLE
Merge in Feat/python-standards

### LINKED ISSUE #2: Python-specific guidelines

### PR DESCRIPTION
**Pull request by kbarone-arch**
Adding in the python standards. Especially interested to hear thoughts about the packages section.

### CODE REVIEW

File: python_standards.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+Inline comments are unnecessary and in fact distracting if they state the obvious. Don’t do this:
+```{python}
+x = x + 1         # Increment x
+```
+But sometimes, this is useful:
+```{python}
+x = x + 1         # Compensate for border
+```
+## Packages
```
Comment (janejuenyang) [inline python_standards.md:167]: Any others for the categories Reggie included in the [R standards](https://github.com/HHS/acf-datasurge-standards/blob/main/r_standards.md)?

File: python_standards.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    return magic(r = real, i = imag)
+
+```
+### Naming
+There are many opinions on the best way to name variables in python. We will generally follow the guidelines in [PEP 8 - Naming Conventions](https://peps.python.org/pep-0008/#naming-conventions)
+#### Naming conventions
+- Variable names should be no less than 3 characters and should be ddscriptive of what the variable represents
+- Use CAPITALS to define constants or globals that will be used in your entire program or module
+- Use lowercase_with_underscores for local variables, member attributes or methods, function arguments and parameters
+- Use PascalCase for classes
```
Comment (janejuenyang) [inline python_standards.md:105]: Is this consistent with https://github.com/HHS/acf-datasurge-standards/blob/main/script.md#case-conventions?

File: python_standards.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
+# Python Standards
+- [Style Considerations](#style-considerations)
```
Comment (janejuenyang) [inline python_standards.md:2]: I like this page-level table of contents!

File: python_standards.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
+# Python Standards
```
Comment (janejuenyang) [inline python_standards.md:1]: Please update the README to include these standards. I think it makes sense for both R and Python to be bullets nested under Scripts in the README.

**Review by janejuenyang — CHANGES_REQUESTED** [review]
Thanks @kbarone-arch! A couple small adjustments, please.

File: python_standards.md (hunk 2 above)
Comment (kbarone-arch) [reply python_standards.md:105]: We should update the script.md file to include a note that global variables should be capitalized and class definitions should use pascal or camel case (at least in python). Thanks for noting this! I'll open an issue.

File: python_standards.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+# No:  
+def complex(real, imag = 0.0):  
+    return magic(r = real, i = imag)
+
+```
+### Naming
+There are many opinions on the best way to name variables in python. We will generally follow the guidelines in [PEP 8 - Naming Conventions](https://peps.python.org/pep-0008/#naming-conventions)
+#### Naming conventions
+- Variable names should be no less than 3 characters and should be ddscriptive of what the variable represents
```
Comment (RGilliard-Arch) [inline python_standards.md]: Minor typo here ddscriptive

File: python_standards.md (hunk 1 above)
Comment (kbarone-arch) [reply python_standards.md:167]: Added in an Assertive Programming section, and packages on geospatial analyses and dashboarding.

**Review by janejuenyang — APPROVED** [review]
Looks great - thanks for making the updates!

**Pull request merged**